custom/icds_reports/templates/icds_reports/icds_app/mobile/map-chart.html

```diff
@@ -1,34 +1,76 @@
 {% load staticfiles %}
-<div>
-  <div class="app-bg h-100vh overflow-hidden" ng-show="!$ctrl.filtersOpen">
```

Comment:
I recommend not really worrying about this file in this commit. all the important bits got changed later